### PR TITLE
update: stable unsigned integer types

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: docs/topics/basic-types.md [hunk 1]
```diff
@@ -316,6 +312,11 @@ Unsigned types are implemented using feature that's not yet stable, namely [inli
 
 #### Unsigned arrays and ranges 
 
+> Unsigned arrays and operations on them are [Beta](components-stability.md). They are almost stable, but migration
```
Comment (koshachy) [inline docs/topics/basic-types.md:315]:
```suggestion
> Unsigned arrays and operations on them are in [Beta](components-stability.md). They are almost stable, but migration
```

File: docs/topics/basic-types.md [hunk 2]
```diff
@@ -325,8 +326,17 @@ Same as for primitives, each of unsigned type has corresponding type that repres
 
 Same as for signed integer arrays, they provide similar API to `Array` class without boxing overhead. 
 
-Also, [ranges and progressions](ranges.md) are supported for `UInt` and `ULong` by classes `UIntRange`,
-`UIntProgression`, `ULongRange`, and `ULongProgression`. 
+When using unsigned arrays, a warning will be reported, indicating that this feature has not been released to stable.
```
Comment (koshachy) [inline docs/topics/basic-types.md:329]: Seems too complicated, mb smth like (can be rephrased): "When you use unsigned arrays, you'll get a warning that indicates that this feature is not stable"?

File: docs/topics/basic-types.md [hunk 3]
```diff
@@ -325,8 +326,17 @@ Same as for primitives, each of unsigned type has corresponding type that repres
...
+To remove the warning, opt in using one of the following ways:
+
+* To propagate the opt-in requirement, annotate declarations that use unsigned integers with  `@ExperimentalUnsignedTypes`.
```
Comment (koshachy) [inline docs/topics/basic-types.md:332]:
```suggestion
* To propagate the opt-in requirement, annotate declarations that use unsigned integers with `@ExperimentalUnsignedTypes`.
```

File: docs/topics/basic-types.md (hunk 2 above)
Comment (p7nov) [reply docs/topics/basic-types.md]: Thanks. Rephased and removed the detailed instruction about opt-in.

**Review by koshachy — APPROVED** [review]

File: docs/topics/basic-types.md [hunk 4]
```diff
@@ -316,6 +312,11 @@ Unsigned types are implemented using feature that's not yet stable, namely [inli
 
 #### Unsigned arrays and ranges 
 
+> Unsigned arrays and operations on them are in [Beta](components-stability.md). They are almost stable, but migration
```
Comment (ilya-g) [inline docs/topics/basic-types.md:315]: This seems to give quite more strict guaranties than it was before:
> It's up to you to decide if your clients have to explicitly opt-in into usage of your API, but keep in mind that unsigned types are not a stable feature, so API which uses them can be suddenly broken by changes in the language.

Comment (p7nov) [reply docs/topics/basic-types.md:315]: We now have standard warning messages for Experimental, Alpha, and Beta features based on descriptions in [Stability of Kotlin components](https://kotlinlang.org/docs/components-stability.html). 
However, I remember that you said unsigned arrays are actually experimental and we just called them Beta before the levels were strictly defined. 
So, if the previous wording reflects the current state accurately, we can use it as a custom warning for this unsigned arrays. Should we?

**Pull request merged**